spec/dummy/config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  root to: redirect("/admin/schools")
```

Comment:
I guess this is not needed as https://github.com/thoughtbot/administrate/pull/1574/files got merged

Reply:
Yeah, we can drop 0585c69 from this branch.